packages/api/src/util/customDecorators.ts

```diff
@@ -1,21 +1,26 @@
 import {Api, ApiRx} from '@cennznet/api';
+import createSubmittable from '@cennznet/api/submittable/createSubmittable';
```

Comment:
same problem, should use relative path. and some old codes have this issue as well, pls correct them together.